packages/amplify-category-auth/src/provider-utils/awscloudformation/import/index.ts

```diff
@@ -355,17 +355,13 @@ const validateUserPool = async (
 ): Promise<boolean | string> => {
   const userPoolClients = await cognito.listUserPoolClients(userPoolId);
   const webClients = userPoolClients.filter(c => !c.ClientSecret);
-  const nativeClients = userPoolClients.filter(c => c.ClientSecret !== undefined);
+  const nativeClients = userPoolClients;
 
   // Check if the selected user pool has at least 1 native and 1 web app client configured.
```

Comment:
Comment is not true anymore, needs update to not to cause confusion for future us